src/msw/spinctrl.cpp

```diff
@@ -404,6 +404,12 @@ wxSpinCtrl::~wxSpinCtrl()
     gs_spinForTextCtrl.erase(GetBuddyHwnd());
 }
 
+void wxSpinCtrl::Refresh(bool eraseBackground, const wxRect *rect)
+{
+    wxControl::Refresh(eraseBackground, rect); 
+    ::RedrawWindow(GetBuddyHwnd(), 0, 0, RDW_INVALIDATE | RDW_UPDATENOW);
```

Comment:
Does it still work without `RDM_UPDATENOW`? We don't use it in the base class `Refresh()`, so it seems strange to use it just for the buddy control and I'd prefer to remove it if things still work without it.

OTOH we could add `RDW_ERASE` if `eraseBackground == true`, again, for consistency with the default implementation.

Reply:
Yes, it works. Fixed.